doc/zmq_getsockopt.txt

```diff
@@ -434,6 +434,20 @@ Default value:: ZMQ_NULL
 Applicable socket types:: all, when using TCP or IPC transports
 
 
+ZMQ_ZMTP_STRICT: Require versioned ZMTP protocol
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
For markup the ~ should match the characters above

Reply:
done